README.md

```diff
@@ -8,61 +8,70 @@ If you want to contribute to this list (please do), send me a pull request.
 
 <!-- MarkdownTOC depth=4 -->
 
-- [Specification](#spec)
```

Comment:
@setchy Could you undo these changes on TOC?

Reply:
Fixed, thanks @chentsulin